pombola/kenya/views_facebook_experiments.py

```diff
@@ -0,0 +1,200 @@
+            recordPositiveResponse(self.survey_session, '62', '10173')
+
+        return super(TreatmentOlympics, self).get(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        context = super(TreatmentOlympics, self).get_context_data(**kwargs)
+        context['sid'] = self.survey_session
+        return context
+
+
+class TreatmentOlympicsThanks(TemplateView):
+
+    template_name = 'facebook-experiment-olympics-thanks.html'
+
+    def get(self, request, *args, **kwargs):
+        survey_session = request.GET.get('sid')
+
+        if survey_session:
+            recordPositiveResponse(survey_session, '58', '10170')
+
+        return super(TreatmentOlympicsThanks, self).get(request, *args, **kwargs)
+
+
+class TreatmentPolitics(TemplateView):
+
+    template_name = 'facebook-experiment-politics.html'
+
+    def get(self, request, *args, **kwargs):
+        self.survey_session = request.GET.get('sid')
+
+        if self.survey_session:
+            recordPositiveResponse(self.survey_session, '61', '10172')
+
+        return super(TreatmentPolitics, self).get(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        context = super(TreatmentPolitics, self).get_context_data(**kwargs)
+        context['form_counties'] = ConstituencyGroupedByCountySelectForm()
+        context['sid'] = self.survey_session
+        return context
+
+
+class TreatmentPoliticsThanks(TemplateView):
+
+    template_name = 'facebook-experiment-politics-thanks.html'
+
+    def get(self, request, *args, **kwargs):
+        survey_session = request.GET.get('sid')
+
+        if survey_session:
+            recordPositiveResponse(survey_session, '59', '10171')
+
+        return super(TreatmentPoliticsThanks, self).get(request, *args, **kwargs)
+
+
+class SurveyGizmoRecord(TemplateView):
+
+    # The survey doesn't actually care if this works or not, just send an "OK".
+    template_name = 'facebook-ajax-response.html'
+
+    def get(self, request, *args, **kwargs):
+        survey_session = request.GET.get('sid')
+        question_id = request.GET.get('qid')
+        selection_id = request.GET.get('selid')
+
+        if survey_session and question_id and selection_id:
+            recordPositiveResponse(survey_session, question_id, selection_id)
+
+        return super(SurveyGizmoRecord, self).get(request, *args, **kwargs)
+
+
+class SurveyGizmoValue(TemplateView):
+
+    # The survey doesn't actually care if this works or not, just send an "OK".
+    template_name = 'facebook-ajax-response.html'
+
+    def get(self, request, *args, **kwargs):
+        survey_session = request.GET.get('sid')
+        question_id = request.GET.get('qid')
+        value = request.GET.get('value')
+
+        if survey_session and question_id and value:
+            recordResponseValue(survey_session, question_id, value)
+
+        return super(SurveyGizmoValue, self).get(request, *args, **kwargs)
+
+
+def getSurveyIdFromSession(survey_session):
```

Comment:
Functions in Python should be lower case, separated by underscores. This function is only used in this module, so ideally you should prefix it with an underscore too. The same applies to the other functions defined below.